src/extension.ts

```diff
@@ -38,43 +38,132 @@ import {
     IMPORT_MOPS_PACKAGE,
     TestParams,
     TestResult,
+    ENVIRONMENT,
 } from './common/connectionTypes';
 import { watchGlob } from './common/watchConfig';
 import { formatDocument } from './formatter';
+import { startReplica } from './server/replicaManager';
+import {
+    getCanisterNames,
+    getCanisterNamesForCandid,
+} from './server/canisterNames';
+import { resolveCandidUIAddress } from './server/candidAddressProvider';
+import { exec } from 'child_process';
+import { TerminalProvider } from './server/terminalProvider';
 
 const config = workspace.getConfiguration('motoko');
 
 let client: LanguageClient;
 
 export function activate(context: ExtensionContext) {
+    const outputChannel = vscode.window.createOutputChannel('ICP logs');
+    const terminalProvider = new TerminalProvider();
+    context.subscriptions.push(outputChannel);
     context.subscriptions.push(
         commands.registerCommand('motoko.startService', () =>
             startServer(context),
         ),
     );
+    // context.subscriptions.push(
+    //     commands.registerCommand(
+    //         'motoko.deployPlayground',
+    //         async (relevantUri?: Uri) => {
+    //             const uri =
+    //                 relevantUri?.toString() ||
+    //                 window.activeTextEditor?.document?.uri.toString();
+    //             if (!uri || !uri.endsWith('.mo')) {
+    //                 window.showErrorMessage(
+    //                     'Invalid deploy URI:',
+    //                     uri ?? `(${uri})`,
+    //                 );
+    //                 return;
+    //             }
+    //             await deployPlayground(context, uri);
+    //         },
+    //     ),
+    // );
+    context.subscriptions.push(
+        commands.registerCommand('motoko.importMopsPackage', async () => {
+            await importMopsPackage(context);
+        }),
+    );
+    context.subscriptions.push(
+        commands.registerCommand('motoko.startReplica', async () => {
```

Comment:
Maybe we could also include a `motoko.stopReplica` command for completeness?

Reply:
ok, i will include it